packages/rich-text/src/helpers/validations.ts

```diff
@@ -30,6 +30,10 @@ const isFormattingOptionEnabled = (field, validationType, nodeTypeOrMark) => {
   // TODO: In the future, validations will always be opt-in. In that case
   // we don't need this step.
   if (enabledFormattings === undefined) {
+    if ([MARKS.SUBSCRIPT, MARKS.SUPERSCRIPT].includes(nodeTypeOrMark)) {
+      return false;
+    }
```

Comment:
What is this for?

Reply:
> sub/sup toolbar options are only shown when the enabledMarks validation explicitly allows them

Follow this AC

Reply:
But that's handled by [this code](https://github.com/contentful/field-editors/blob/5fee0717a38a1c83a518ddcf117624fe452f8bfa/packages/rich-text/src/Toolbar/index.tsx#L83). The logic for explicit marks vs implicit is handled in Content apps e.g. web app & studio. In other words, the rich-text field editor should always receive a list of explicit enabled marks/nodes in the `field.validation` attribute.

Reply:
Okay cool, thank you for clarifying!